src/faq/index.md

```diff
@@ -2,6 +2,7 @@
 title: FAQ
 type: faq
 layout: page
+toc: true
```

Comment:
the large font-size is good for readability, but for the TOC to take up more of the entire page seems a bit odd. perhaps we can file an issue to look into putting it as a sidebar - or at least allow folks to toggle it à la Wikipedia's TOCs:

![image](https://cloud.githubusercontent.com/assets/203725/16472567/d9e4fde0-3e19-11e6-805e-89c38095bd9d.png)